<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMusic</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(255, 214, 184);
}
QListWidget
{
padding: 10px;
border-radius: 35px;
border : 2px solid black;
background : rgb(247, 235, 255);
}
QListWidget QScrollBar
{
background : lightblue;
}
 QListView::item:selected
{
border : 1px solid black;
background : rgb(173, 153, 255);
}
 QListView::item
{
margin:3 px;
}
QSlider::groove:horizontal
{  
height: 10px;
margin: 0px;
border-radius: 5px;
background: rgb(0, 0, 0);
}
QSlider::handle:horizontal {
background: rgb(240, 255, 190);
border: 1px solid #E3DEE2;
width: 17px;
margin: -5px 0; 
border-radius: 8px;
}
QSlider::sub-page:qlineargradient 
{
background: rgb(204, 133, 18);
border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>781</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="playlist_name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Мне нравится</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>562</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>590</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 30px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>play.png</normaloff>play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="playing_time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>565</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="duration_time">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>565</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>595</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(77, 77, 77);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>next.png</normaloff>next.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>595</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(77, 77, 77);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>prev.png</normaloff>prev.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="add_track_but">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px; 
border: 2px groove gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>add_track.png</normaloff>add_track.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>29</width>
      <height>29</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="make_playlist_but">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px; 
border: 2px groove gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>add_playlist.png</normaloff>add_playlist.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>29</width>
      <height>29</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="repeat_but">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>590</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: None;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>repeat1.png</normaloff>repeat1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="open_playlist">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: None;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>explorer.png</normaloff>explorer.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="length">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Всего:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>595</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(77, 77, 77);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>plus.png</normaloff>plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="minus10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>595</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(77, 77, 77);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>minus.png</normaloff>minus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
